src/pages/en/core-concepts/framework-components.md

```diff
@@ -141,7 +141,7 @@ This allows you to build entire "apps" in your preferred JavaScript framework an
 
 ## Customize Your Frameworks
 
-By default, Astro ships with support out-of-the-box for React, Preact, Svelte and Vue. You can configure Astro to add new frameworks or remove support for any that you are not using.
+Astro ships with support for React, Preact, Svelte and Vue. You can configure Astro to add new frameworks.
```

Comment:
I was just looking at this statement again this morning, and I think it's difficult to say what is or isn't *included* by default. I looked through a bunch of our starter templates and each one has different renderers included.

But, they are all *supported* (by default!) as long as the renderer is listed. ie you don't have to install any packages, "import React from react" or anything like that. So, in my mind, it *is* correct to say that they are supported by default. But, that in order to *use* them, (in order to use only the dependencies you'll actually need in your project) you'll need to edit your list of available renderers... which includes adding and removing them, I think!

I'm open to perhaps a better way of expressing all this? What do you think?  :)

Reply:
Sorry I think I should have clarified, this PR is in preparation of removing the renderers as direct dependencies of `astro`. So besides having to put used renderers in the config they also need to be installed as a dependency by the developer/user.

Reply:
The examples include the used renderer(s) in their dependencies, I recently added those in this PR: https://github.com/withastro/astro/pull/2727